### PR TITLE
Merge master branch into develop

### LINKED ISSUE #505: Merge master branch into develop
Currently develop is behind master, so it would be nice to have them synchronized.

### PR DESCRIPTION
**Pull request by ManuelJNunez**
### Summary

Merge master into develop.

### Details and comments

close #505

### Checks

-   [x] I have updated the documentation accordingly.
-   [x] I have read the [CONTRIBUTING](https://github.com/rticommunity/rticonnextdds-examples/blob/develop/CONTRIBUTING.md) document.

### CODE REVIEW

**Comment by lulivi** [comment]
@ManuelJNunez can you add the latest changes in master to this pr?

**Review by fgaranda — APPROVED** [review]

**Comment by ManuelJNunez** [comment]
Added last changes from master.

File: examples/connext_dds/CMakeLists.txt [hunk 1]
```diff
@@ -67,7 +67,7 @@ connextdds_add_examples_subdirectories(
         "custom_flow_controller"
         "custom_transport"
         "deadline_contentfilter"
-        "discovery_snapshot"
+        # "discovery_snapshot"
```
Comment (fgaranda) [inline examples/connext_dds/CMakeLists.txt:70]: This one should be enabled in develop I think.

**Review by fgaranda — APPROVED** [review]

**Comment by alexcamposruiz** [comment]
This PR is no longer relevant.

**Pull request closed without merging**